webui/src/css/sass/app.scss

```diff
@@ -0,0 +1,110 @@
+@import "./variables";
+@import "./mixins";
+
+@import url('https://fonts.googleapis.com/css?family=Nunito:400,600,700&display=swap');
```

Comment:
the fonts must be embedded into Traefik.

Reply:
done:
https://github.com/containous/traefik/pull/5249/commits/b09fe49df97f00923a5e4606f034fffa67c9966d